python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -13,3 +13,31 @@ private import semmle.python.dataflow.new.DataFlow
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import semmle.python.dataflow.new.TaintTracking
 private import experimental.semmle.python.Frameworks
+
+module NoSQLQuery {
+  abstract class Range extends DataFlow::Node {
+    abstract DataFlow::Node getQueryNode();
+  }
+}
+
+class NoSQLQuery extends DataFlow::Node {
+  NoSQLQuery::Range range;
+
+  NoSQLQuery() { this = range }
+
+  DataFlow::Node getQueryNode() { result = range.getQueryNode() }
```

Comment:
As I briefly [mentioned](https://github.com/github/codeql/pull/5680#discussion_r616002031) in the other pr, I think this should be `getQuery` (to match what the rest of the naming convention we've used in the non-experimental Concepts.qll)